spec/controllers/users_controller_spec.rb

```diff
@@ -160,4 +160,15 @@
       expect(user.locale).to eq('es')
     end
   end
+
+  describe '#show' do
+    context 'when user not found' do
+      before {allow(controller).to receive(:current_user).and_return(nil) }
```

Comment:
This is unnecessary. In this scope, since we didn't do anything like `allow(controller).to receive(:current_user).and_return(user)`, `current_user` will be `nil` anyway (which means visiting the page while logged out).